### PR TITLE
Add citation recommendation for Pymagicc

### LINKED ISSUE #110: Add citation recommendation for Pymagicc paper
- cite MAGICC6 paper
- cite Pymagicc paper
- cite archived version

### PR DESCRIPTION
**Pull request by rgieseke**
Closes #110

### CODE REVIEW

**Comment by rgieseke** [comment]
Need to check the docs rendering, some headings broke by this.

**Comment by rgieseke** [comment]
This also adds @znicholls  as an email contact :-]

File: README.rst [hunk 1]
```diff
@@ -251,32 +251,54 @@ Contributing
...
     M. Meinshausen, S. C. B. Raper and T. M. L. Wigley (2011). "Emulating coupled
     atmosphere-ocean and carbon cycle models with a simpler model, MAGICC6: Part I
     "Model Description and Calibration." Atmospheric Chemistry and Physics 11: 1417-1456.
-    `doi:10.5194/acp-11-1417-2011 <https://dx.doi.org/10.5194/acp-11-1417-2011>`_
+    `https://doi.org/10.5194/acp-11-1417-2011 <https://dx.doi.org/10.5194/acp-11-1417-2011>`_
 
-See also the `MAGICC website <http://magicc.org/>`_ and
-`Wiki <http://wiki.magicc.org/index.php?title=Main_Page>`_
-for further information.
+If you use Pymagicc in your research, please additionaly cite
```
Comment (znicholls) [inline README.rst]: @rgieseke typo, 'additionaly' needs two 'l''s

Otherwise merge :)

Comment (rgieseke) [reply README.rst]: Good catch! Will fix!

**Pull request merged**